.github/workflows/pull_request.yaml

```diff
@@ -53,3 +53,55 @@ jobs:
           files: ./coverageunit.out
           verbose: true
           fail_ci_if_error: false
+
+  bench:
```

Comment:
🪑 ? 🤣

Reply:
It's `go test -bench` so you know 😆

Reply:
jeez i hate Go authors right now

Reply:
My two cents.. We're only saving a few characters. I'd recommend we just be explicit and say `benchmark` or `go-bench` (my preferred recommendation).

Reply:
That also works for me. Updated. Thanks!